nservicebus/messaging/databus/index.md

```diff
@@ -95,16 +95,20 @@ Automatically removing these attachments can cause problems in many situations.
 
 ## Alternatives
 
-- Use a different transport or a different transport tier
-- Message Compression: Use message body compression which works well on text-based payloads like XML and Json or any payload (text or binary) that contains repetitive data
-  - [Message mutator example demonstrating message body compression](/samples/messagemutators/)
-- Stream-based properties: The [Handling large stream properties via pipeline](/samples/pipeline/stream-properties/) sample demonstrates a purely stream-based approach (rather than loading the full payload into memory) implemented by leveraging the NServiceBus pipeline.
-- Binary Serializer: A binary serializer is more efficient and most serializers can be added with a few lines of code
-   - Some binary [serializers are maintained by the community](/nservicebus/community/#serializers)
-- Attachments: When dealing with unbounded binary payloads consider the [community maintained NServiceBus.Attachments](/nservicebus/community/#nservicebus-attachments)
-  - Read on demand: Will only retrieve attachment data when the consumer reads it
-  - Reduced Memory usage: No base64 serializer overhead resulting in a significant reduction in resource utilization
-- Use any of the above in combination with compression
+NOTE: A combination of these techniques may be used.
+
+- Use a different transport or different tier (e.g. Azure Service Bus _Premium_ instead of _Standard_).
+- Use message body compression, which works well on text-based payloads like XML and JSON or any payload (text or binary) that contains repetitive data.
+  - The [message mutator sample](/samples/messagemutators/) demonstrates message body compression.
+- Use stream-based properties.
+  - The sample showing how to [handle large stream properties via the pipeline](/samples/pipeline/stream-properties/) demonstrates a purely stream-based approach (instead of loading full payloads into memory).
+- Use a more efficient serializer, such as a binary serializer.
+  - A custom serializer can usually be implemented in only a few lines of code.
+  - Some binary [serializers are maintained by the community](/nservicebus/community/#serializers).
+- Use [NServiceBus.Attachments](/nservicebus/community/#nservicebus-attachments) for unbounded binary payloads. The package is similar to the Data Bus but has some differences:
+  - Read on demand: Attachments are only retrieved when read by a consumer.
+  - Async enumeration: The package supports processing all data items using an `IAsyncEnumerable`.
+  - Reduced memory usage: The package avoids the use of a base64 serializer which results in a significant reduction in memory usage.
```

Comment:
_continued from https://github.com/Particular/docs.particular.net/pull/6199/files#r1372702758_

```suggestion
  - No base64 serialization: This results in a significant reduction in memory usage.
  - Direct stream access: This makes the package more suitable for [binary large objects (BLOBs](https://en.wikipedia.org/wiki/Binary_large_object) since stream contents do not necessarily have to be loaded into memory before storing them or when retrieving them.
```

@ramonsmits where is base64 serialization used? I.e. a significant reduction in memory usage compared to what?

Reply:
@adamralph In the serializers when dealing with byte arrays. ALL that happens in memory and if you are dealing with binary streams to begin with then there is no need for all of this overhead.

Reply:
```suggestion
  - No serialization: The serializer is not used, which may result in a significant reduction in memory usage.
  - Direct stream access: This makes the package more suitable for [binary large objects (BLOBs](https://en.wikipedia.org/wiki/Binary_large_object) since stream contents do not necessarily have to be loaded into memory before storing them or when retrieving them.
```